lib/flame_k8s_backend/runner_pod_template.ex

```diff
@@ -178,6 +178,17 @@ defmodule FLAMEK8sBackend.RunnerPodTemplate do
         do: [],
         else: object_references(parent_pod_manifest)
 
+    parent = FLAME.Parent.new(parent_ref, self(), FLAMEK8sBackend, runner_pod_name, "POD_IP")
```

Comment:
@chrismccord
I had to add this hack in order to be able to test the Livebook integration. Maybe `FLAME.Parent.new/5` could at some point allow to pass the `backend_vrs`.

and while at it the variable could also be renamed to `backend_version` so us normalo humans don't have to reverse engineer it 😜